htdocs/eventorganization/conferenceorbooth_card.php

```diff
@@ -493,8 +493,17 @@
 	//$keyforbreak='fieldkeytoswitchonsecondcolumn';	// We change column just before this field
 	//unset($object->fields['fk_project']);				// Hide field already shown in banner
 	//unset($object->fields['fk_soc']);					// Hide field already shown in banner
-	include DOL_DOCUMENT_ROOT.'/core/tpl/commonfields_view.tpl.php';
+	$link_subscription = 'http://localhost/dolibarr/htdocs/public/eventorganization/attendee_subscription.php?id=';
+	//$link_subscription .= base64_encode($id);
+	$key = 'DV3PH';
+	$link_subscription .= dol_encode($id, $key);
+	$link_subscription .= '&securekey='.$conf->global->EVENTORGANIZATION_SECUREKEY;
```

Comment:
Use
$link_subscription .= '&securekey='.urlencode($conf->global->EVENTORGANIZATION_SECUREKEY);
so if param was corrupted it is not possible to disturb the url forging.